styx-api-service/src/main/java/com/spotify/styx/api/StatusResource.java

```diff
@@ -161,7 +162,7 @@ private static String arg(String name, RequestContext rc) {
   }
 
   private Map<WorkflowInstance, RunState> getActiveStates(String componentsStr) throws IOException {
-    var components = Arrays.asList(componentsStr.split(","));
+    var components = new HashSet<>(Arrays.asList(componentsStr.split(",")));
```

Comment:
A better way to do it is:

```java
var components = Set.of(componentsStr.split(","));
 ```

Reply:
No this won't work unfortunately because `Set.of` fails on duplicates.